packages/round-manager/src/features/round/ViewRoundStats.tsx

```diff
@@ -161,7 +163,7 @@ export default function ViewRoundStats() {
 }
 
 type StatsCardProps = {
-  text: string | number;
+  text?: string | number;
   title: string;
   grayBorder?: boolean;
```

Comment:
I think instead of using a boolean value, we could use a string prop named borderColor to make it more flexible for future use:
```
type StatsCardProps = {
  text?: string | number;
  title: string;
  borderColor?: "gray" | "violet";
};
```

Reply:
if we need to style the border colors further, we can pass-through classNames later, rn no requirement to do this so will keep as-is